inference-engine/src/gna_plugin/gna_graph_compiler.cpp

```diff
@@ -1205,7 +1224,7 @@ void GNAGraphCompiler::EltwisePrimitive(InferenceEngine::CNNLayerPtr layer) {
     uint32_t num_rows_in = in_4b_channels * in_4b_height * in_4b_width;
     uint32_t num_columns_in = in_4b_batch;
     uint32_t num_rows_out = num_rows_in;
-    uint32_t num_padding = ALIGN(num_rows_in, 8) - num_rows_in;
+    uint32_t num_padding = ALIGN(num_rows_in, inputsSignificance) - num_rows_in;
```

Comment:
The "significance" in variable name is taken from the ALIGN macro parameter name. Here I would have to use a generic name (not connected with 8-bits or 16-bits precisions) for the "rounding value". It could be noOfInputsRoundingValue or something similar. And for 8-bit GNALimitations it would be quite lengthy - noOfInputsRoundingValueLowPrec. What do you think?

Reply:
IMO Factor, Divider, MultipleOf
eg., noOfInputsFactor

but it is not so important